src/metadata/metadata_handler.cc

```diff
@@ -112,7 +113,7 @@ void MetadataHandler::extractMetaData(const std::shared_ptr<Context>& context, c
 #endif
 
 #ifdef HAVE_FFMPEG
-    if (contentType != CONTENT_TYPE_PLAYLIST && ((contentType == CONTENT_TYPE_OGG && item->getFlag(OBJECT_FLAG_OGG_THEORA)) || startswith(item->getMimeType(), "video") || startswith(item->getMimeType(), "audio"))) {
+    if (contentType != CONTENT_TYPE_PLAYLIST && ((contentType == CONTENT_TYPE_OGG && item->getFlag(OBJECT_FLAG_OGG_THEORA)) || itemCls == UPNP_CLASS_MUSIC_TRACK || itemCls == UPNP_CLASS_VIDEO_ITEM)) {
```

Comment:
This can be reduced to

``if (contentType != CONTENT_TYPE_PLAYLIST && (itemCls == UPNP_CLASS_MUSIC_TRACK || itemCls == UPNP_CLASS_VIDEO_ITEM))``

Reply:
Would a the itemCls checks on their own even be enough?

Reply:
> Would a the itemCls checks on their own even be enough?

If mime type to itemCls is complete as it is with default config. 

Makes everything clearer: we set item class in the beginning and then handle files according to their class. At the moment esp. the check for ogg is done very often - only to confirm that video is handled as video.